### PR TITLE
Dcgan mx print

### PR DESCRIPTION
**Pull request by StevenJokes**
*Description of changes:*

By submitting this pull request, I confirm that you can use, modify,
copy, and redistribute this contribution, under the terms of your
choice.

### CODE REVIEW

**Comment by StevenJokes** [comment]
http://preview.d2l.ai/d2l-en/PR-1393/chapter_generative-adversarial-networks/dcgan.html
image
image

**Comment by StevenJokes** [comment]
Maybe our DCGAN have extra nn.BatchNorm2d(ndf)? :https://github.com/d2l-ai/d2l-en/issues/1381
